### PR TITLE
Added KeyExists

### PR DESCRIPTION
**Pull request by lesichkovm**

### CODE REVIEW

File: map.go [hunk 1]
```diff
@@ -1,5 +1,14 @@
 package lo
 
+// KeyExists returns wheter the given key exists.
+// Play: https://go.dev/play/p/bAq9mHErB4V
```
Comment (Weihao1101) [inline map.go:4]: The Playground mismatch this function

**Comment by lesichkovm** [comment]
@samber can this be merged please?

**Comment by lesichkovm** [comment]
@samber ?

**Comment by samber** [comment]
I'm going to rename it to `lo.HasKey` before merge, to match the go standards.

**Pull request closed without merging**